components/raftstore/src/store/fsm/peer.rs

```diff
@@ -947,12 +947,21 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
     fn on_extra_message(&mut self, msg: &RaftMessage) {
         let extra_msg = msg.get_extra_msg();
         match extra_msg.get_type() {
-            ExtraMessageType::MsgRegionWakeUp => {
-                self.reset_raft_tick(GroupState::Ordered);
+            ExtraMessageType::MsgRegionWakeUp | ExtraMessageType::MsgCheckStalePeer => {
+                if self.fsm.group_state == GroupState::Idle {
```

Comment:
Why the change?

Reply:
In one test, I find the leader looks stopped because the `chech-peer-stale-tick` is too frequent and sends too much `MsgCheckStalePeer` to others which causes others `register_raft_base_tick` so I add a check here to avoid useless wakeup.